plugins/lang/mapsmenu.txt

```diff
@@ -217,8 +217,8 @@ CHANGLE_MENU = Verander Map Menu
 [es]
 RESULT_REF = Resultado rechazado
 RESULT_ACC = Resultado aceptado
-VOTE_SUCCESS = Votacion finalizada. Se va a cambiar el mapa a
-VOTE_FAILED = Votacion sin exito
+VOTE_SUCCESS = Votacion finalizada (Tiene: %d) (Necesario: %d) Se va a cambiar el mapa a '%s'.
```

Comment:
French translation is also weird. I think you should just add only the translations you know for sure. The missing keys can be dealt later in a global language update or something.